tests/designs/vhdl_zerovector/Makefile

```diff
@@ -0,0 +1,31 @@
+# Simulator configuration
```

Comment:
Please add the following header to all new files:
```suggestion
# Copyright cocotb contributors
# Licensed under the Revised BSD License, see LICENSE for details.
# SPDX-License-Identifier: BSD-3-Clause

# Simulator configuration
```

Reply:
Addressed in 040dae2ef19d7a6b951cda011860c6004d36c003

Reply:
Thanks, but the VHDL and Python files as well please.